### PR TITLE
Add org list

### LINKED ISSUE #2307: Organizations tab for admins
### Discussed in https://github.com/woodpecker-ci/woodpecker/discussions/2257

<div type='discussions-op-text'>

<sup>Originally posted by **lonix1** July 10, 2023</sup>
### Clear and concise description of the problem

The server config page shows tabs: "Secrets", "Users", "Agents" and "Queue".

There should be another tab, "Organizations".

There are various org-level settings, e.g. organization secrets (exposed by the API), but they aren't viewable in the UI. So they are effectively "hidden" and it's easy to forget about it.

### Suggested solution

A new tab named "Organizations".

Doesn't need to be complicated, or to allow interaction. Only to show the orgs, and secrets related to them. Readonly to keep it simple.

### Alternative

_No response_

### Additional context

_No response_

### Validations

- [X] Checked that the feature isn't part of the `next` version already [https://woodpecker-ci.org/faq#which-version-of-woodpecker-should-i-use]
- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't already an [issue](https://github.com/woodpecker-ci/woodpecker/issues) that request the same feature to avoid creating a duplicate.</div>

### PR DESCRIPTION
**Pull request by qwerty287**
Screenshot 2023-08-28 at 10-08-20 Woodpecker
Closes #2307

### CODE REVIEW

File: pipeline/rpc/proto/woodpecker.pb.go
[the remark this answers is not in the document]
Comment (anbraten) [reply pipeline/rpc/proto/woodpecker.pb.go:1]: @qwerty287 I will open a separate PR. (Strange when I ran that command in #2173 it did nothing for me 🙈 )

File: server/store/store.go [hunk 1]
```diff
@@ -189,6 +189,7 @@ type Store interface {
 	OrgFindByName(string) (*model.Org, error)
 	OrgUpdate(*model.Org) error
 	OrgDelete(int64) error
+	GetOrgList(*model.ListOptions) ([]*model.Org, error)
```
Comment (anbraten) [inline server/store/store.go:192]:
```suggestion
	OrgList(*model.ListOptions) ([]*model.Org, error)
```

**Review by anbraten — APPROVED** [review]

**Comment by xoxys** [comment]
@qwerty287 Do I miss something?

`{"source":"https://github.com/woodpecker-ci/woodpecker","version":"next-479ced3b25"}`

image

image

**Comment by xoxys** [comment]
```
 id |    name    | is_user | private 
----+------------+---------+---------
  1 | woodpecker | t       | f
  2 | testorg    | t       | f
(2 rows)
```

But it's an org in gitea :thinking:

image

**Comment by xoxys** [comment]
Added a new org in Gitea now with a new repo. Can enable it but WP but still not shown and `is_user=true`...

```
  3 | testorg2   | t       | f
```

Looks like a bigger issue to me.

**Comment by xoxys** [comment]
Created an issue to track it.

**Pull request merged**